chat_client/src/chat_receiver.py

```diff
@@ -0,0 +1,60 @@
+import logging
+import threading
+from typing import Iterator
+
+import grpc
+import protobufs.chat_pb2 as chat_pb2
+
+
+class ChatReceiver(threading.Thread):
+    """Thread class which listen for incomming messages till stop(). The thread itself has to check
+    regularly for the stopped() and set unauth condition on rpc_error UNAUTHENTICATED."""
```

Comment:
`"""` should be at new string when docstring is multilined.

Reply:
Done